.github/workflows/test_models.yaml

```diff
@@ -139,7 +139,7 @@ jobs:
           fi
           sudo tar -xzf ${CACHED_DOWNLOAD_DIR}/geckodriver-v${GECKO_VER}-linux64.tar.gz -C /usr/local/bin
           geckodriver --version
-          pip install --upgrade selenium
+          pip install --upgrade selenium==4.2.0
```

Comment:
We need this because the new selenium on pypi (4.3.0) just broke other imports (see slack discussion; michael has a PR in to selenium request project). Even if we fix this on baselayer, the `--upgrade` will try to push selenium to the broken version.

Reply:
We should remove the upgrade flag, at least.

Reply:
Done.